docker/hive2-ldap/Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM keboola-hive2-no-auth
 
+# Import test data
+RUN BEELINE_ARGS="-u 'jdbc:hive2://localhost:10000'" /tmp/data/data-init.sh
```

Comment:
Testovacie data sa musia importovat az nakoniec, inak to nefungovalo spravne, ... preto je to tu presunute.